### PR TITLE
Add a note about jmx remote connections

### PR DESCRIPTION
**Pull request by MishaDemianenko**

### CODE REVIEW

**Comment by MishaDemianenko** [comment]
[[S4][InnovoComerce] unable to get local JMX metrics when started as service](https://trello.com/c/0iNJyTWJ/8294-s4innovocomerce-unable-to-get-local-jmx-metrics-when-started-as-service)

File: modules/ROOT/pages/jmx-metrics.adoc [hunk 1]
```diff
@@ -166,6 +166,9 @@ Connect to the process running your Neo4j database instance:
 .Connecting JConsole to the Neo4j Java process
 image::jconsole_connect1.png[alt="Connecting with JConsole", width=300]
 
+Please note: when you are connecting to a remote process or, in some cases, local that is running as a service please choose a `Remote Process` connection option.
```
Comment (renetapopova) [inline modules/ROOT/pages/jmx-metrics.adoc:169]:
```suggestion
[NOTE]
====
When connecting to a remote process or, in some cases, a local process running as a service, you must choose the `Remote Process` connection option.

The `<port>` value is configured by the `com.sun.management.jmxremote.port` property.
====
```

File: modules/ROOT/pages/jmx-metrics.adoc [hunk 2]
```diff
@@ -166,6 +166,9 @@ Connect to the process running your Neo4j database instance:
...
+<port> is value that is configured by `com.sun.management.jmxremote.port` property.
```
Comment (renetapopova) [inline modules/ROOT/pages/jmx-metrics.adoc:170]:
```suggestion
```

**Review by renetapopova — APPROVED** [review]
Thanks, @MishaDemianenko! Some editorial suggestions.

**Comment by MishaDemianenko** [comment]
Hi, @renetapopova I have a question, how does cherry-picking work in this repo? automatic or should I create that for 4.4 manually? Thx in advance!

**Comment by renetapopova** [comment]
Hi @MishaDemianenko, cherry-picking is manual.

**Comment by renetapopova** [comment]
@MishaDemianenko, I am happy to do it.

**Comment by MishaDemianenko** [comment]
Oh nice, thank you so much :bow: !

**Comment by renetapopova** [comment]
No probs!

**Pull request merged**